front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -228,7 +228,9 @@ export const usePathfinding = (
 
           if (
             pathfindingResult.status === 'success' ||
-            pathfindingResult.status === 'incompatible_constraints'
+            (pathfindingResult.status === 'failed' &&
+              pathfindingResult.failed_status === 'not_found' &&
+              pathfindingResult.error_type === 'incompatible_constraints')
```

Comment:
Since you use this condition 2 times, you can create a `const` for it, and use this instead